src/Apps/CollectorProfile/Routes/Saves2/Components/StackedImageLayout.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Image } from "@artsy/palette"
+import { prepareImageURLs } from "Apps/CollectorProfile/Routes/Saves2/Utils/prepareImageURLs"
+import { FC } from "react"
+import { cropped } from "Utils/resized"
+
+interface StackedImageLayoutProps {
+  imageURLs: (string | null)[]
+}
+
+interface StackImageProps {
+  url: string | null
+  index: number
+}
+
+const LARGE_IMAGE_SIZE = 190
+const SMALL_IMAGE_SIZE = 112
```

Comment:
I don't like how specific these numbers are. Did design think about if these things scale at all? or how they relate to the grid?

Reply:
Clarified this question [here](https://www.figma.com/file/wSV2KNz97Tyj8HkH2ZYAkB?node-id=308:216953#357768111)

Reply:
I don't think there will be any problems when scaling, since we specify specific [sizes](https://github.com/artsy/force/blob/42feb5c09093d6dff3a857e7b44dac9206a459a7/src/Apps/CollectorProfile/Routes/Saves2/Components/SavesItem.tsx#L30-L31) for the rail item

Reply:
Yeah, I think this is fine. I guess in general I'm trying to understand if there's any underlying systematic logic. A lot of the layouts we get have elements size seemingly at random, whereas obviously there should be some site-wide consistency to how we approach this.